cli/EAP7-924_Coloured_output_for_CLI.adoc

```diff
@@ -0,0 +1,83 @@
+= EAP7-924 Coloured output for CLI
+:author:            Ingo Weiss
+:email:             ingo@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+The intent of this feature is to provide a better user experience by means of colouring important parts of the CLI workflow, this includes error messages, warning messages, successful outputs and better visual feedback when using batch and flow control mechanisms if/for/while.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}EAP7-924[EAP7-924]
+
+=== Related Issues:
+
+* {issue-base-url}WFCORE-3450
+* {issue-base-url}WFCORE-3577
+
+=== Dev Contacts:
+
+* mailto:ingo@redhat.com[Ingo Weiss]
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts:
+
+* TBD
+
+=== Affected Projects or Components:
+
+* WildFly Core CLI
+
+
+== Requirements
+
+=== Hard requirements
```

Comment:
Do you plan to use colours if cli script is used? (The answer needs to be mentioned in this proposal)

```
[mkopecky@dhcp-10-40-4-239 bin]$ ./jboss-cli.sh --file=script.cli 
a
[mkopecky@dhcp-10-40-4-239 bin]$ ./jboss-cli.sh < script.cli
You are disconnected at the moment. Type 'connect' to connect to the server or 'help' for the list of supported commands.
[disconnected /] echo a
a
[disconnected /] [mkopecky@dhcp-10-40-4-239 bin]$ 
```

Both previous ways are used by users for automation and colours could break user's automation scripts for evaluation of CLI output.

Reply:
It is, atm, disabled by default and I think it should be that way, especially for scripts.